modules/targetVideoBidAdapter.js

```diff
@@ -181,7 +183,7 @@ function getBannerHtml(vastUrl) {
     <body>
       <div id="targetVideoPlayer"></div>
       <script src="https://player.target-video.com/custom/targetvideo-banner.js"></script>
-      <script>initPlayer("${vastUrl}");</script>
+      <script>initPlayer('${vastUrl}', '${bidParamsJSON}');</script>
```

Comment:
Also, it would help to destructure the `bidParams` into the information that is necessary to be passed to your initPlayer function. In this case it would be one property, `placementId`.